crates/ruff_workspace/src/options.rs

```diff
@@ -455,6 +455,9 @@ pub struct LintOptions {
     )]
     pub exclude: Option<Vec<String>>,
 
+    /// Options for the `ruff` plugin
+    pub ruff: Option<RuffOptions>,
```

Comment:
```suggestion
    /// Options for the `ruff` plugin
    #[option_group]
    pub ruff: Option<RuffOptions>,
```

Our options macros are a bit complicated. Especially because you have to add both serde and non serde attributes. Anyway, adding `#[option_group]` should make the group to appear in the documentation

Reply:
That did the trick, thank you!